test/functional/homepage_controller_test.rb

```diff
@@ -36,11 +36,23 @@ class HomepageControllerTest < ActionController::TestCase
           </div>
         </div>
         EOF
+      end
 
-        get :index
-        assert_template "homepage/_promo_banner"
-        assert @response.body.include?("homepage-promo-banner")
-        assert @response.body.include?("Some Title")
+      context "with an emergency banner enabled" do
+        should "not render the promo banner" do
+          EmergencyBanner.any_instance.stubs(:enabled?).returns(true)
+          get :index
+          refute @response.body.include?("homepage-promo-banner")
+        end
+      end
+
+      context "with an emergency banner not enabled" do
+        should "render promo banner" do
+          get :index
```

Comment:
Shouldn't the EmergencyBanner be explicitly set to `false`? Similar to the set up of the previous test?